sdk/helper/pluginutil/runner.go

```diff
@@ -98,6 +99,8 @@ func CtxCancelIfCanceled(f context.CancelFunc, ctxCanceler context.Context) chan
 	return quitCh
 }
 
+// MultiplexingSupport returns true if a plugin supports multiplexing.
+// Currently this is hardcoded for database plugins.
 func MultiplexingSupport(version int) bool {
```

Comment:
This will likely need a Plugin Type arg in the future but I think it is ok to leave out for now.